man/systemd.network.xml

```diff
@@ -2525,6 +2525,18 @@ NFTSet=prefix:netdev:filter:eth_ipv4_prefix</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Bootp=</varname></term>
```

Comment:
given we uppercase `[DHCP]` section name we should probably uppercase BOOTP= setting name too